Cards/Include/Ultimate.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <iostream>
+
+class Ultimate
+{
+  public:
+         virtual std::string useUlt() = 0;
+};
+
+class Ultimate_Mage_Race : public Ultimate
+{
+  public:
+          int heal;
+          int dmg;
+          Ultimate_Mage_Race(int = 8,int = 2);
```

Comment:
To samo, co pisałem w review u @MikolajOgan, IMO lepiej napisać co poszczególne parametry znaczą